test/integration/helpers.js

```diff
@@ -2,12 +2,12 @@
 
 const escapeRegExp = require('escape-string-regexp');
 const os = require('os');
-const fs = require('fs-extra');
+const fs = require('fs');
+const fsExtra = require('fs-extra');
```

Comment:
```suggestion
const {
  utimesSync,
} = require('fs');
const {
  ensureDirSync,
  readFileSync,
} = require('fs-extra');
```

Thinking out loud: How about destructuring here?

Also: Doesn't `fs-extra` re-export everything from `fs`? What's the reason for introducing `fs` in addition to `fs-extra`?

Reply:
i didnt know it is re exporing. 
I want to make as few as possible changes.

PTAL